cms/static/coffee/src/keyword_validator.coffee

```diff
@@ -0,0 +1,28 @@
+define ->
```

Comment:
It's the same code as used on the LMS side, but since I've got his vanilla javascript for the CMS checked out now I'm not using this anymore